default.py

```diff
@@ -58,6 +58,14 @@ def sortResultsByRes(fetched_urls):
                   utils.parse_resolution_of_source(x[0]),
                   reverse=True)
 
+#Will be called at handle_player
+def on_percent():
+    percent_bool = True if 'true' in control.getSetting('watchlist.percentbool') else False
```

Comment:
You can register callback or pass None depends on setting instead of always pass the callback and check inside :)